### PR TITLE
Disable function size by default ##breaking

### PR DESCRIPTION
**Pull request by ITAYC0HEN**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)

**Detailed description**

Until now, when disassembling a function or the top-part of it, Rizin will print the size of the function near the signature.
For example "44" in the first line here:

```
┌ 44: int main (char **argv);
│           ; var int32_t var_4h @ ebp-0x4
│           ; arg char **argv @ esp+0x24
│           0x000011cd      lea   ecx, [argv]
│           0x000011d1      and   esp, 0xfffffff0
 . . . 
 . . .
```

This pull request introduces a new configuration variable named `asm.fcn.size` that will be disabled by default.
In addition, it changes the name of the `asm.fcnsig` to `asm.fcn.signature` because Developers' lingo isn't users' lingo.

**Test plan**

disassemble using `pdf` and `pdf` as well; as graph, and observe how the `asm.fcn.size` changes when disabled and enabled.

### CODE REVIEW

**Review by ret2libc — APPROVED** [review]

**Review by kazarmy — APPROVED** [review]

**Review by Maijin — APPROVED** [review]

**Review by XVilka — CHANGES_REQUESTED** [review]
I suggest to convert only half of the tests to the default output, so we test both modes, just in case. Apart from that LGTM.

**Comment by Maijin** [comment]
No I would leave like this, but add a test for each options, just copy/pasta and add the eval and change the name. Tests are supposed to test one and one feature only, not multiple one's 👍

**Comment by XVilka** [comment]
@Maijin majority of tests are "integration" tests since involve many aspects of the Rizin, especially in `analysis` category.

**Comment by Maijin** [comment]
yeah but again, I think it'd be better to just copy/paste one or few of those and add an eval true/false to them to test both  functionnalities instead separately.

**Comment by officialcjunior** [comment]
And, I'll handle documenting this new variable on the book. 👍🏽

File: test/db/cmd/cmd_pd [hunk 1]
```diff
@@ -642,7 +642,7 @@ e asm.flags=false
 s sym.func.100004401
 af
 afn newname
-shu
+sHu
```
Comment (ITAYC0HEN) [inline test/db/cmd/cmd_pd]:
```suggestion
```

**Comment by ITAYC0HEN** [comment]
Would love help with fixing the last failing tests. Not sure what's going on in there

**Comment by XVilka** [comment]
@ITAYC0HEN you forgot to add a test with the size enabled. Please do this in a separate PR.

File: test/db/cmd/cmd_pd [hunk 2]
```diff
@@ -675,7 +675,38 @@ e asm.lines.fcn=false
...
+CMDS=<<EOF
+e asm.bytes=true
+e asm.lines.fcn=true
+s 0x004017c0
+af+ 0x004017c0 fcn2.0x004017c0
+afb+ 0x4017c0 0x4017c0 13 0x4017dd 0x4017cd
+pdf
+?e
+e asm.lines.fcn=false
+e asm.fcn.size=true
```
Comment (ITAYC0HEN) [inline test/db/cmd/cmd_pd:705]: @XVilka I did here
I can add a few more of needed

Comment (XVilka) [reply test/db/cmd/cmd_pd:705]: Ah, didn't notice. No worries then, should be fine.

**Pull request merged**